tutormfe/templates/mfe/build/mfe/env/development

```diff
@@ -0,0 +1,15 @@
+DISCOVERY_API_BASE_URL={% if DISCOVERY_HOST is defined %}http://{{ DISCOVERY_HOST }}:8381{% endif %}
```

Comment:
This file should be deleted now that we have dynamic config, right? (same for production file below)

Reply:
Apologies - of course!  This got past me on the rebase. 😬